notebooks/EDA-and-modeling.ipynb

```diff
@@ -16,412 +16,451 @@
+      "Dimensions of hydroponics_lettuce_N_53.png: 1024 x 1003\n",
+      "Dimensions of hydroponics_lettuce_N_47.png: 1020 x 999\n",
+      "Dimensions of hydroponics_lettuce_N_46.png: 1024 x 1001\n",
+      "Dimensions of hydroponics_lettuce_N_52.png: 1020 x 1000\n",
+      "Dimensions of hydroponics_lettuce_N_56.png: 1023 x 1006\n",
+      "Dimensions of hydroponics_lettuce_N_42.png: 1024 x 1005\n",
+      "Dimensions of hydroponics_lettuce_N_43.png: 1023 x 1006\n",
+      "Dimensions of hydroponics_lettuce_N_57.png: 1024 x 1006\n",
+      "Dimensions of hydroponics_lettuce_N_41.png: 1023 x 1005\n",
+      "Dimensions of hydroponics_lettuce_N_55.png: 1024 x 1003\n",
+      "Dimensions of hydroponics_lettuce_N_54.png: 1024 x 1003\n",
+      "Dimensions of hydroponics_lettuce_N_40.png: 1024 x 1003\n"
+     ]
     }
-   },
-   "outputs": [],
+   ],
    "source": [
-    "from sklearn.model_selection import train_test_split\n",
+    "from PIL import Image\n",
+    "import os\n",
     "\n",
-    "X_train, X_test, y_train, y_test = train_test_split(X, Y, test_size=0.20, random_state=42)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "slideshow": {
-     "slide_type": "skip"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "#dropping Quakers column and unnamed\n",
-    "#changing one of the altitude to log and droping the original\n",
-    "X_train[\"altitude_mean_log\"] = np.log(X_train[\"altitude_mean_meters\"])\n",
-    "X_train.drop(['altitude_mean_meters'], axis=1, inplace=True)\n",
-    "X_train.drop(['Quakers'], axis=1, inplace=True)\n",
-    "X_train.drop(['Unnamed: 0'], axis=1, inplace=True)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "slideshow": {
-     "slide_type": "skip"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "X_train.info()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "slideshow": {
-     "slide_type": "skip"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "altitude_low_meters_mean = X_train[\"altitude_low_meters\"].mean()\n",
-    "altitude_high_meters_mean = X_train[\"altitude_high_meters\"].mean()\n",
-    "altitude_mean_log_mean = X_train[\"altitude_mean_log\"].mean()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "slideshow": {
-     "slide_type": "skip"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "# fillna with mean.. \n",
-    "X_train[\"altitude_low_meters\"] = X_train[\"altitude_low_meters\"].fillna(altitude_low_meters_mean)\n",
-    "X_train[\"altitude_high_meters\"] = X_train[\"altitude_high_meters\"].fillna(altitude_high_meters_mean)\n",
-    "X_train[\"altitude_mean_log\"] = X_train[\"altitude_mean_log\"].fillna(altitude_mean_log_mean)"
+    "# Specify the folder path containing the images\n",
+    "folder_path = \"/Users/ouzibigouziouzi/Desktop/FNNPK/-N\"\n",
```

Comment:
You want to keep it like this? Than when combining snippets, responsible person has to adjust the path.

Reply:
i would change it to '/path/to/input/images' to standarize it, what @richiwang and @Linn2202 think?

Reply:
that seems a good solution for now, I will search a way of not hardcoding the path, if there is one